Requirements for SSE.md

```diff
@@ -79,17 +79,17 @@ TODO: Insert Diagram photo
 
-#### TODO: Misuse case Name
+#### Third-Party Add-ons Misuse Case
 
-TODO: Insert info here
+Joel is a hacker who likes to mess with others networks and devices. Joel creates a Home Assistant add-on that supports a very specific IoT device that has no official Home Assistant add-on support. He creates the add-on to run on the hosts network and require elevated system rights to run properly. He also adds a backdoor to the add-on code for him to gain control of the network and devices of users who download his add-on. Home Assistant limits add-ons with rights to run on the host network to be able to directly communicate with all other add-ons by name. This allows Joel to mess with Jack's network and connected IoT devices he has set up with Home Automation.
 
 #### Prevention/Security Requirement
 
-TODO: Assess alignment of security requirements derived from mis-use case analysis with advertised features of the open-source software. Review OSS project documentation and codebase to support your observations. Summarize your observations.
+Risks of installing third-party add-ons is difficult for Home Assistant to control. Though they provide many warnings to the user that they cannot guarantee the quality or security of third-party add-ons. Home Assistant also rates third-party add-ons for users to be able to distinguish which third-party add-ons they recommend and don't recommend using with Home Assistant. All add-ons added to Home Assistant initially run on protection enabled mode which prevents add-ons from getting rights on the system, but Home Assistant allows users to disable this if add-ons require more rights but warns users that this could destroy their system. Home Assistant also uses an internal network that allows communication between add-ons by using names or alias so they are able to defend against outside threats. Home Assistant Community repository requires contributors to contact owners to discuss any contributions before adding an add-on to the repository, and requires two other developers to approve before the change is merged into the repository.
```

Comment:
So a couple of things I noticed, for the security prevention you can also hit that HomeAssistant takes detailed logs of all addons and what they do, addons are basically docker containers and inside their own individual container, and only communicate with http://hassio/homeassistant/api/services/
Also dont forget AppArmor and discuss more of the "Protection enabled mode" because that is one of the key software features HA has to prevent the misuse case here.

Reply:
I saw that they took logs of all add-ons but i didn't think it would really prevent "Jack" from installing third party add-ons and more like monitor add-ons once installed and see if something isn't working right.
I revised and included the discussion of the add-on docker containers and add-on protection mode.
For AppArmor, HA doesn't add this themselves to the thirdpart add-ons, they actually provide it to the developers to add themselves if they want to.

Reply:
That makes sense. It seems they off other features but leave it up to the developer to enable it. Yeah logging is more of a detective control rather than a preventive one. Other than that document looks good the Portction-enabled-mode and each adding being a separate container are good controls plus it will prevent the misuse case